### ISSUE TITLE
fatal: Not a git repository (or any of the parent directories): .git

### ISSUE THREAD

**Issue by huangjj27** [rust-native-ui/libui-rs#10]
reduced log as follow:
```bash
$ RUST_BACKTRACE=1 cargo check
   Compiling ui-sys v0.1.1
error: failed to run custom build command for `ui-sys v0.1.1`
process didn't exit successfully: `C:\Users\34937\Desktop\learning-rust\iui-example\target\debug\build\ui-sys-b9529245fd369183\build-script-build` (exit code: 101)
--- stderr
fatal: Not a git repository (or any of the parent directories): .git
thread 'main' panicked at '
command did not execute successfully, got: exit code: 1

build script failed, must exit now', C:\Users\34937\.cargo\registry\src\mirrors.ustc.edu.cn-61ef6e0cd06fb9b8\cmake-0.1.31\src\lib.rs:643:5
```

then, I search for the [build.rs:L10-L12](https://github.com/LeoTindall/libui-rs/blob/97be7aeca50dd044274c1cd73c2979b28e8cdb20/ui-sys/build.rs#L10):
```rust
// Update the submodule with libui if needed
    if !Path::new("libui/.git").exists() {
        Command::new("git").args(&["submodule", "update", "--init"]).status().unwrap();
}
```

I'm wondering why it gave hints of "Not a git repository"？ What happenned when I ran cargo?

**Comment by huangjj27** [comment]
bug in #2. the issue is closing

**Comment by NoraCodes** [comment]
FYI this is currently fixed (?) in a branch, so when that gets merged I'll ping you here

**Comment by NoraCodes** [comment]
Recent push should have fixed this, please retest

**Comment by huangjj27** [comment]
retest failed. see https://github.com/LeoTindall/libui-rs/issues/2#issuecomment-391568260

**Comment by NoraCodes** [comment]
Should be resolved by #14
